src/Aquifer.Data/EventHandlers/BadTranslationPairHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Aquifer.Data.Entities;
+using Aquifer.Data.Exceptions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace Aquifer.Data.EventHandlers;
+
+public static class BadTranslationPairHandler
+{
+    private static readonly IReadOnlyList<string> s_KeyBlacklist = ["span", "div"];
```

Comment:
Others to consider: `p`, `h1`

Reply:
I think we're gonna disallow anything below two characters. I'll just blanket disallow it.